app/js/controllers/themes.js

```diff
@@ -0,0 +1,29 @@
+/* global React, ReactDOM */
+
+import { Controller } from 'components/fxos-mvc/dist/mvc';
+
+import Themes from 'js/views/themes';
+import ThemesNew from 'js/views/themes-new';
+
+export default class ThemesController extends Controller {
+  main(action = '') {
```

Comment:
My plan is to use the default case as the route for viewing a specific recipe like:
```
#themes/recipe-id
```

Would that be better to default the action parameter to `list` and rename the case in the switch?